R/utils.R

```diff
@@ -140,3 +140,51 @@ resolve_modules_datanames <- function(modules, datanames, join_keys) {
     modules
   }
 }
+
+check_modules_datanames <- function(modules, datanames) {
```

Comment:
```suggestion
#' Check `datanames` in modules
#' 
#' This function ensures specified `datanames` in modules match those in the data object, 
#' returning error messages or TRUE for successful validation.
#'
#' @param modules (`teal_modules`) object
#' @param datanames (`character`) names of datasets available in the `data` object
#'
#' @return A character vector containing error messages or TRUE if validation passes.
#' @keywords internal
check_modules_datanames <- function(modules, datanames) {
```
Adding documentation text.